Source/WebCore/rendering/RenderLayerModelObject.cpp

```diff
@@ -486,6 +488,30 @@ RenderSVGResourceClipper* RenderLayerModelObject::svgClipperResourceFromStyle()
     return nullptr;
 }
 
+RenderSVGResourceFilter* RenderLayerModelObject::svgFilterResourceFromStyle() const
+{
+    if (!document().settings().layerBasedSVGEngineEnabled())
+        return nullptr;
+
+    const auto& operations = style().filter();
+    if (operations.size() != 1)
```

Comment:
Don't we have isEmpty? ;-)

Reply:
Here I want to check whether a single referenced SVG filter or not. So first I check if number of operations is 1 and then I check that this operation is `ReferenceFilterOperation`